### PR TITLE
literaturesuggest: various fixes in thesis form

### PR DESCRIPTION
**Pull request by Koursaros93**
Removed defense date and reordered the fields as asked in issue #438
Made Phd default degree type.

### CODE REVIEW

**Comment by jmartinm** [comment]
Please rebase this PR with latest `master` branch to see if tests are passing.

File: inspirehep/bat/pages/create_literature.py [hunk 1]
```diff
@@ -131,24 +131,24 @@ def _thesis_info_population(input_data):
...
-    )
     Arsenic().find_element_by_id('institution').send_keys(
         input_data['institution']
     )
+    Arsenic().find_element_by_id('supervisors-0-name').send_keys(
+        input_data['supervisor']
+    )
+    Arsenic().find_element_by_id('supervisors-0-affiliation').send_keys(
+        input_data['supervisor-affiliation']
+    )
```
Comment (jacquerie) [inline inspirehep/bat/pages/create_literature.py]: So, here you are reordering the selections according to the actual form order. This is not wrong, but is unrelated with what you are doing. When that happens, you should split the related changes in their own commits. For example here you have a commit that reorders things around, and one that deletes `defense_date`.

File: inspirehep/modules/literaturesuggest/forms.py [hunk 2]
```diff
@@ -439,15 +439,10 @@ def _is_other_language(language):
...
-    defense_date = fields.TextField(
-        label=_('Date of Defense'),
-        description='Format: YYYY-MM-DD, YYYY-MM or YYYY.',
-        validators=[date_validator],
-        widget=defensedate_widget,
-    )
 
     degree_type = fields.SelectField(
         label=_('Degree Type'),
+        default="phd",
```
Comment (jacquerie) [inline inspirehep/modules/literaturesuggest/forms.py]: And this is a third commit.

File: tests/acceptance/test_literature_suggestion_form.py [hunk 3]
```diff
@@ -48,12 +48,11 @@ def test_literature_create_thesis_manually(login):
...
         'report-number-1': '101',
-        'supervisor': 'Mister Yellow',
-        'supervisor-affiliation': 'CERN',
-        'thesis-date': '2001-01-01',
-        'thesis-defense': '2001-01-01',
         'degree-type': 'bachelor',
+        'thesis-date': '2001-01-01',
         'institution': 'Wisconsin U., Madison',
+        'supervisor': 'Mister Yellow',
+        'supervisor-affiliation': 'CERN',
```
Comment (jacquerie) [inline tests/acceptance/test_literature_suggestion_form.py]: Same comment as above: you are reordering for clarity (this is a dictionary, so the order does not matter!), so you should be precise about it.
